src/test/java/com/checkout/payments/RequestApmPaymentsIT.java

```diff
@@ -404,4 +409,90 @@ void shouldMakeStcPayPayment() {
             assertTrue(exception.getCause() instanceof CheckoutApiException);
         }
     }
+
+    @Test
+    void shouldMakeAlmaPayment() {
+        final PaymentRequest paymentRequest = PaymentRequest.builder()
+                .source(RequestAlmaSource.builder()
+                        .billingAddress(TestHelper.createAddress())
+                        .build())
+                .currency(Currency.EUR)
+                .amount(10L)
+                .capture(true)
+                .successUrl("https://testing.checkout.com/sucess")
+                .failureUrl("https://testing.checkout.com/failure")
+                .build();
+
+        checkPayeeNotOnboarded(() -> paymentsClient.requestPayment(paymentRequest));
+    }
+
+    @Test
+    void shouldMakeKlarnaPayment() {
+        final PaymentRequest paymentRequest = PaymentRequest.builder()
+                .source(RequestKlarnaSource.builder()
+                        .accountHolder(TestHelper.getAccountHolder())
+                        .build())
+                .currency(Currency.EUR)
+                .amount(10L)
+                .capture(true)
+                .successUrl("https://testing.checkout.com/sucess")
+                .failureUrl("https://testing.checkout.com/failure")
+                .build();
+
+        try {
+            paymentsClient.requestPayment(paymentRequest).get();
+            fail();
+        } catch (Exception exception) {
+            assertTrue(exception.getCause() instanceof CheckoutApiException);
```

Comment:
Maybe we should validate the error message, so that we know that the exception is the one we are expecting?